### DISCUSSION TITLE
Steam isn't installed

### DISCUSSION THREAD

**Discussion by dnkmmr69420** [Jovian-Experiments/Jovian-NixOS#188] [Q&A]
I tested this in a vm and the gaming-session does not run and steam isn't installed

**Comment by samueldr** [comment]
Hi,

I'm not trying to imply "RTFM" or anything like that. I've noticed the current section about configuration was lacking information about our improvements. So I'm asking you to look at it assuming you already looked at it and are confused about the lack of `steam`, and the Steam Deck UI not launching.

Would the changes in #132 help you understand what is going on here?

I'll also add a note here that I'm not sure if the gamescope-based Steam Deck UI session should or shouldn't work with whatever VM setup you're using. So that may be why the gaming session doesn't work. You may want to look at `journalctl` to see if there's anything about that.

If you still need some help with that issue, please share at least the full `jovian.*` configuration options from your NixOS config, or if you can, your whole config.

**Comment by dnkmmr69420** [comment]
does it automatically install steam or do you need to use the steam module as well.

**Comment by samueldr** [comment]
When you import the module, other than the overlay being added (and it should be behind an option too I guess), the whole project is a no-op.

This is because you might want to only enable the Steam Deck hardware things, but are not interested in using Steam at all, or might want to use the Steam Deck UI, but not on a steam deck.

So the defaults are to change nothing, but give out options.

**Comment by dnkmmr69420** [comment]
the steam deck ui doesn't work because lib-virt doesn't support vulkan

**Comment by samueldr** [comment]
Though, to be crystal clear, when I say "you", it's the general "anyone" you.

And who would want to enable that UI but not steam deck stuff? Someone not using a Steam Deck!

We have at least two NVIDIA desktop computer users who are using some of the features here, and the goal is that if anyone had a fancy non-Steam Deck controller-form-factor computer (e.g. ROG Ally) it should be possible to add their quirks in this repo.

**Comment by dnkmmr69420** [comment]
ok good to know
